system/Validation/Rules.php

```diff
@@ -65,6 +65,21 @@ public function differs(string $str = null, string $field, array $data): bool
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Equals the static value provided.
+	 *
+	 * @param string $str
+	 * @param string $str
```

Comment:
Please correct the param here. It should be $val

Reply:
Thanks for catching that! Fixed.